demos/01-reading-users/Helpers/AuthHandler.cs

```diff
@@ -0,0 +1,24 @@
+using System.Net.Http;
```

Comment:
Where is this class used? I've eyeballed the code and I don't see it. I went back and re-read the learn module where you create the project and see that it tells you to create it, but I still don't see where it's used?